client/packages/coldchain/src/Equipment/DetailView/StatusForm.tsx

```diff
@@ -64,9 +86,10 @@ const Row = ({
   );
 };
 
-export const StatusTab = ({ draft, onChange, value }: AssetLogPanel) => {
+export const StatusForm = ({ draft, onChange }: StatusForm) => {
   const t = useTranslation();
   const isGaps = useIsGapsStoreOnly();
+  const isMobile = useIsScreen('sm') && Capacitor.getPlatform() === 'android';
```

Comment:
Also checking for is native because photo button won't work on web view for mobile

Reply:
In other places we're using this. Do you think it would suit here too?
`Capacitor.isNativePlatform()`

Reply:
Thanks James, and I agree! 

The issue in this case was that the Upload component  (called from the StatusForm component) uses `isMobile = useIsScreen('sm')` in stead of an is native check.

I was a bit wary of duplicating this component when I imagine there will be a future refactor would maybe change how we manage mobile vs tablet UI.

I am thinking in this case it might be easier to make a separate component for native (mobile and tablet) upload in this case?